lib/Command/Room/TRoomCommand.php

```diff
@@ -314,6 +314,20 @@ protected function removeRoomParticipants(Room $room, array $userIds): void {
 		}
 	}
 
+	protected function removeUsersFromAllRooms(array $userIds): void {
+		$users = [];
+		foreach ($userIds as $userId) {
+			$user = $this->userManager->get($userId);
+			if (!$user) {
+				throw new InvalidArgumentException(sprintf("User '%s' not found.", $userId));
+			}
+			$users[] = $user;
+		}
+		foreach ($users as $user) {
+			$this->manager->removeUserFromAllRooms($user);
+		}
+	}
```

Comment:
if you don't need anything else from the TRoomCommand, I would say don't use it and just put the code directly into your Command class